pkg/restore/restore.go

```diff
@@ -1258,6 +1258,10 @@ func (ctx *restoreContext) restoreItem(obj *unstructured.Unstructured, groupReso
 			case internalVolume.PodVolumeBackup:
 				restoreLogger.Infof("Dynamically re-provisioning persistent volume because it has a pod volume backup to be restored.")
 				ctx.pvsToProvision.Insert(name)
+				// add restore warning if user is trying to restore from snapshots but the backup uses fs backup  as default option
+				if boolptr.IsSetToTrue(ctx.backup.Spec.DefaultVolumesToFsBackup) && boolptr.IsSetToTrue(ctx.restore.Spec.RestorePVs) {
+					warnings.Add(namespace, errors.Errorf("restorePVs should not be set as true when backup consists of defaultVolumesToFSbackup as true and all volumes are using FS Backup method. RestorePVs specifies whether to restore all included PVs from snapshot"))
```

Comment:
I would suggest a minor change in the message as follows (or something similar):

"Ignoring restorePVs option since the backup method was file system backup. This flag  is only valid for snapshots."

BTW. Is this going to be logged once for each PV or is it a global message for entire restore?

Reply:
I'm not sure we can say "ignoring" here just based on the default flag. This particular volume could be using snapshots even if that flag is set.